test/linkedList/LinkedListTest.java

```diff
@@ -161,5 +158,58 @@ public void testAddEntryToTail()
       result = testLL.addEntryToTail(testNode3);
       assertTrue(result == 0);
    }
-
+   
+   /**
+    * Test method for {@link linkedList.LinkedList#toString()}.
+    */
+   @Test
+   public void testToString()
+   {
+      String expectedOutput = "This list of size 10 contains  \n" +
+                              "Data: 109\n" +
+                              "Data: 108\n" +
+                              "Data: 107\n" +
+                              "Data: 106\n" +
+                              "Data: 105\n" +
+                              "Data: 104\n" +
+                              "Data: 103\n" +
+                              "Data: 102\n" +
+                              "Data: 101\n" +
+                              "Data: 100\n";
+      int result;
+      /* create a list of 10 elements */
+      LinkedList someList = new LinkedList();
+      Integer[] objectArray = {100, 101, 102, 
+                               103, 104, 105, 
+                               106, 107, 108, 
+                               109};
+      ListNode[] nodeArray = new ListNode[10];
+      for(int ii = 0; ii < objectArray.length; ii++)
```

Comment:
This is something I picked up from Shaun Wakumoto 9 years ago. ii is still a short variable name and it is unique in that I can search for it - how many words do you know with "ii" in it?